chromium_src/components/payments/content/payment_request.cc

```diff
@@ -0,0 +1,113 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "components/payments/content/payment_request.h"
+
+#include "base/bind.h"
+#include "brave/browser/brave_rewards/rewards_service_factory.h"
+#include "brave/components/brave_rewards/common/constants.h"
+#include "chrome/browser/profiles/profile.h"
+#include "third_party/blink/public/mojom/payments/payment_request.mojom.h"
+
+using payments::mojom::PaymentErrorReason;
+
+#define Init Init_ChromiumImpl
+#include "../../../../../components/payments/content/payment_request.cc"
+#undef Init
+
+namespace payments {
+
+void PaymentRequest::GetPublisherDetailsCallback(
+    mojo::PendingRemote<mojom::PaymentRequestClient> client,
+    std::vector<mojom::PaymentMethodDataPtr> method_data,
```

Comment:
Should this be a const reference?

Reply:
Or maybe that's just how the Chromium interface is defined...

Reply:
This interface is needed for call to `PaymentRequest::Init`, build will fail if we mark `method_data` as const - https://github.com/brave/brave-core/pull/7557/files#diff-12d68f7aee6cfd7305f59bcf92a37b1fa2353b1bd0a8b5224c7f4b63c68bfd2bR37